### DISCUSSION TITLE
Trajectory/Pseudotime with multiple multiome samples.

### DISCUSSION THREAD

**Discussion by mossconfuse** [stuart-lab/signac#1653] [Q&A]
Hi Tim

Is there any hope for a decent trajectory analysis pipeline for Seurat/Signac objects that could handle both batch corrections (Harmony) and work with Multimodal samples (RNA/ATAC)? The available methods seem to be limited to either single sample analysis (scvi, multivelo) or only have one modality (monocle3). Switching between different packages also results in inconsistencies when UMAPs are generated, so a comprehensive package extending directly from Signac would be great.

**Comment by timoast** [comment]
Monocle3 uses the UMAP embedding, so it can work with multimodal data if some other approach (for example, WNN) is used to create a multimodal UMAP embedding first.
